### DISCUSSION TITLE
Where is `mint.navigate`?

### DISCUSSION THREAD

**Discussion by nilslindemann** [mint-lang/mint#607] [Answered] [answered]
Hello,

[`Window.navigate()`](https://github.com/mint-lang/mint/blob/2a4cffcaae76fb8c34dab02effb27fb885b060e0/core/source/Window.mint#LL180C1-L180C1) calls [`_navigate()`](https://github.com/mint-lang/mint/blob/2a4cffcaae76fb8c34dab02effb27fb885b060e0/src/compilers/top_level.cr#L244), which is an alias for `mint.navigate`. Where is this defined? I can not find it anywhere in the source tree.

**Comment by Sija** [comment, marked as answer]
https://github.com/mint-lang/mint-runtime/blob/afe495559915c980f6e82cda5cfd9ea33d01d16b/source/Utils.js#L14-L29

**Comment by nilslindemann** [comment]
Ah, that is why I couldn't find it, it is in another Repo!

*forked*

**Comment by nilslindemann** [comment]
Further question, how does Mint know the code from this repo, when I do "mint start"?

**Reply by Sija** [reply]
It's bundled together with Mint, see here: https://github.com/mint-lang/mint/blob/2a4cffcaae76fb8c34dab02effb27fb885b060e0/src/assets/runtime.js

**Reply by nilslindemann** [reply]
Thank you (again). I also found [the build command in the makefile](https://github.com/mint-lang/mint-runtime/blob/afe495559915c980f6e82cda5cfd9ea33d01d16b/Makefile#L2). I have never before run such a file, would I guess correctly, it is `make default`?

**Reply by gdotdesign** [reply]
Yes `make default` works or just `make` (it will run the first defined task by default).

**Reply by nilslindemann** [reply]
Thanks again.
